### PR TITLE
Handle file being sent as multipart form

### PR DESCRIPTION
**Pull request by rsachdevch**
__Short description outlining key changes/additions__
Handle file being sent as multipart form

__JIRA Ticket Number__
BI-9997

### Type of change

* [ ] Bug fix
* [x] New feature
* [ ] Breaking change

### Pull request checklist

* [x] I have added unit tests for new code that I have added
* [x] I have added/updated functional tests where appropriate
* [x] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/go.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/golang/go/wiki/CodeReviewComments)__

### CODE REVIEW

File: handlers/bulk_refunds.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"net/http"
+
+	"github.com/companieshouse/chs.go/log"
+)
+
+// HandleBulkRefund accepts a bulk refunds file and adds and updates
+// refunds data to the DB
+func HandleBulkRefund(w http.ResponseWriter, req *http.Request) {
+
+	log.InfoR(req, fmt.Sprintf("start POST request for bulk refunds"))
```
Comment (emead) [inline handlers/bulk_refunds.go:14]: No need to use `fmt.Sprintf`

File: handlers/bulk_refunds.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+	_, _, err := req.FormFile("file")
+	if err != nil {
+		log.ErrorR(req, fmt.Errorf("error reading form from request: %s", err))
```
Comment (emead) [inline handlers/bulk_refunds.go:18]: Use `%w` for errors

File: handlers/bulk_refunds.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusInternalServerError)
```
[the remark this answers is not in the document]
Comment (jphillips2121) [reply handlers/bulk_refunds.go:20]: We don't seem to be doing it for any other endpoints in this repo but we'll definitely need to implement it when the web app starts calling it. 
Don't mind if we add in the functionality here or for that particular story.

File: handlers/bulk_refunds.go [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/companieshouse/chs.go/log"
+)
+
+// HandleBulkRefund accepts a bulk refunds file and adds and updates
+// refunds data to the DB
+func HandleBulkRefund(w http.ResponseWriter, req *http.Request) {
```
Comment (jphillips2121) [inline handlers/bulk_refunds.go:12]: Rename this to make it GovPay specific, we might make it generic for both payment methods when we implement paypal but separate endpoints is the way we're planning on doing it originally

File: utils/http.go [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+	return &ResponseResource{Message: message}
+}
+
+// WriteJSONWithStatus writes the interface as a json string with the supplied status.
+func WriteJSONWithStatus(w http.ResponseWriter, r *http.Request, data interface{}, status int) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(status)
+	err := json.NewEncoder(w).Encode(data)
+	if err != nil {
+		log.ErrorR(r, fmt.Errorf("error writing response: %v", err))
```
Comment (emead) [inline utils/http.go:27]: %w for errors

File: utils/http.go [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+	Message string `json:"message"`
+}
+
+// NewMessageResponse - convenience function for creating a response resource
+func NewMessageResponse(message string) *ResponseResource {
+	return &ResponseResource{Message: message}
+}
+
+// WriteJSONWithStatus writes the interface as a json string with the supplied status.
+func WriteJSONWithStatus(w http.ResponseWriter, r *http.Request, data interface{}, status int) {
```
Comment (emead) [inline utils/http.go:22]: Need a unit test for this function.

Comment (emead) [inline utils/http.go:22]: It looks like this function is now being used in a few services. Would it be best to now move it into chs.go?

**Review by emead — APPROVED** [review]

**Pull request merged**